src/python/turicreate/test/test_object_detector.py

```diff
@@ -230,6 +230,12 @@ def test_create_with_empty_dataset(self):
         with self.assertRaises(_ToolkitError):
             tc.object_detector.create(self.sf[:0])
 
+    def test_create_with_fixed_random_seed(self):
+        model_1 = tc.object_detector.create(self.sf, max_iterations=3, random_seed=86)
+        pred_1 = model_1.evaluate(self.sf)
+        model_2 = tc.object_detector.create(self.sf, max_iterations=3, random_seed=86)
+        pred_2 = model_2.evaluate(self.sf)
+        self.assertEqual(pred_1, pred_2)
```

Comment:
Would comparing the outputs of `predict` rather than `evaluate` be a better test?

Reply:
Good!
I can do it.